### ISSUE TITLE
Add Factorio-like beacons

### ISSUE THREAD

**Issue by MineMaarten** [MineMaarten/PneumaticCraft#540]
They basically apply upgrades to a certain area around them.

>The beacons also would be interesting, but I doubt the usefulness of them, or the predictability. In Factorio, apart from power usage, not much can go wrong when machines unnoticeably get affected by the beacon. However, in PneumaticCraft this often isn't the case. Speed upgrades can cause regulation system which didn't prepare for this to break, or temperatures can get too high, causing the efficiency of a compressor to drop dramatically. Another thing is that apart from the Speed and Security Upgrade, not many machines are the affected the same with the same upgrade.
But on the other hand I can see uses of it in arrays of compressors, to let them all get sped up at once... So I'll see.

**Comment by Forecaster** [comment]
Maybe add anti-upgrades that counter the effect of certain upgrades, that way by making a beacon you could apply upgrades to multiple machines (cheaper) while excluding certain machines you don't want the upgrades to apply to.

This may be needlessly complicated, but I thought I'd throw it out anyway.

**Comment by MineMaarten** [comment]
Nah, I like people having to take the consequences of applying upgrades in a whole area. I do want to make clear which blocks are getting affected though so they won't be suprised ;)
